AppInspector.CLI/Writers/AnalyzeHtmlWriter.cs

```diff
@@ -53,8 +53,8 @@ public override void WriteResults(Result result, CLICommandOptions cLICommandOpt
         private void WriteHtmlResult()
         {
             //Grab any local css and js files that are needed i.e. don't have hosted URL's or are proprietary
-            string allCSS = "<style>\n" + MergeResourceFiles(Utils.GetPath(Utils.AppPath.basePath) + "html\\resources\\css");
-            string allJS = "<script type=\"text/javascript\">\n" + MergeResourceFiles(Utils.GetPath(Utils.AppPath.basePath) + "html\\resources\\js");
+            string allCSS = "<style>\n" + MergeResourceFiles(Utils.GetPath(Utils.AppPath.basePath) + "html/resources/css");
```

Comment:
Should this use Path.Combine instead of literal slashes?  That will automatically add the correct slashes for the platform.

Reply:
totally, in a hurry to fix.  hang on.